### PR TITLE
Add editor for coach profile

### PR DESCRIPTION
**Pull request by ExxZzyy**
Add editor for coach profile

### CODE REVIEW

File: exadel-trainee-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -5,7 +5,6 @@ import { UsersComponent } from './components/mock-component/users/users.componen
 import { ManageComponent } from './components/mock-component/manage/manage.component';
 import { CheckComponent } from './components/mock-component/check/check.component';
 import { StatisticsComponent } from './components/mock-component/statistics/statistics.component';
-import { EditorComponent } from './components/mock-component/editor/editor.component';
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/app-routing.module.ts:8]: delete it from mock-component folder

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor.component.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  tabs: CoachEditorTabs[] = [
+    CoachEditorTabs.grammar,
+    CoachEditorTabs.audition,
+    CoachEditorTabs.writingAndSpeaking,
+  ];
+
+  tabChanged(tabChangeEvent: MatTabChangeEvent): void {
+    if (tabChangeEvent.index === 0) {
+      this.selectedTab = 'Grammar Question';
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor.component.ts:37]: its better to store this info in constant object with key CoachEditorTabs and value CoachEditorTabsTitles

**Review by RomPavel — APPROVED** [review]

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor-table/coach-profile-editor-table.module.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { AngularMaterialCommonModule } from '../../../angular-material-common.module';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule, AngularMaterialCommonModule, MatIconModule],
+  exports: [],
+})
+export class CoachProfileEditorTableModule {}
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor-table/coach-profile-editor-table.module.ts:11]: strange module, without any components

File: exadel-trainee-app/src/app/app-routing.module.ts (hunk 1 above)
Comment (ExxZzyy) [reply exadel-trainee-app/src/app/app-routing.module.ts]: Done
dbe5aa64d522d795e19e1e1f7dd162d515f18b4e

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor-table/coach-profile-editor-table.module.ts (hunk 3 above)
Comment (ExxZzyy) [reply exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor-table/coach-profile-editor-table.module.ts]: Done
dbe5aa64d522d795e19e1e1f7dd162d515f18b4e

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor.component.ts (hunk 2 above)
Comment (ExxZzyy) [reply exadel-trainee-app/src/app/models/coach-profile/coach-profile-editor/coach-profile-editor.component.ts]: Done
cdcfdccce8f6bad8077fa8faae21654322b321df

**Pull request merged**